notes-agent/sync_all.sh

```
@@ -0,0 +1,37 @@
+# Cuando el token expire, actualiza ESE archivo con uno nuevo (DevTools → AccessToken).
+
+set -e
+DIR="$(cd "$(dirname "$0")" && pwd)"
+TOKEN_FILE="$HOME/.hidock/hinotes_token"
+LOG="$HOME/.hidock/sync.log"
+
+if [ ! -f "$TOKEN_FILE" ]; then
+  echo "❌ Falta $TOKEN_FILE con tu AccessToken de HiNotes." | tee -a "$LOG"
+  exit 1
+fi
+export HINOTES_TOKEN="$(tr -d '[:space:]' < "$TOKEN_FILE")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)